.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.
+
+Please answer the following:
+
+Code Changes:
+- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
```

Comment:
Maybe we should list some of the things from the Wiki here?

Reply:
go for it

Reply:
I'd rather them not get out of sync which is why I just listed the wiki article as a reference.